src/app/pages/ArticlePage/index.test.tsx

```diff
@@ -869,4 +876,69 @@ describe('Article Page', () => {
       );
     });
   });
+
+  describe('when rendering an AMP page', () => {
+    const pageDataWithSecondaryColumn = {
+      ...articleDataNews,
+      secondaryColumn: {
+        topStories: topStoriesList,
+        features: [],
+      },
+    };
+
+    const renderAmpPage = ({
+      service,
+      pathname,
+    }: {
+      service: Services;
+      pathname: string;
+    }) => {
+      return render(
+        <Context isAmp service={service} pathname={pathname}>
```

Comment:
Replaced `pathname` with `id`. I wasn't too sure about this comment originally since a default isn't set here, could you clarify what you mean sorry?

Reply:
Now that you are using `id` instead of `pathname`, this comment is no longer valid. 

(But just to clarify - within the experiment code you were parsing & splitting the pathname - removing .amp from the end in order to get the id. So what I meant here was, as well as passing `isAmp` = true, should the pathname have also needed `.amp` added to the end so that the code to extract the id from the pathname would work as expected).